### PR TITLE
Fix Error 414 | Request-URI Too Large URL: /admin/object/get-select-o…

### PR DESCRIPTION
**Pull request by mhgtlc**
Fixes issue: https://github.com/pimcore/admin-ui-classic-bundle/issues/218

### CODE REVIEW

**Comment by mhgtlc** [comment]
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/pimcore/pimcore/blob/master/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> _**I have read the CLA Document and I hereby sign the CLA**_
> 
> You can retrigger this bot by commenting **recheck** in this Pull Request


I have read the CLA Document and I hereby sign the CLA

**Comment by mhgtlc** [comment]
recheck

File: src/Controller/Admin/DataObject/DataObjectController.php [hunk 1]
```diff
@@ -551,7 +551,7 @@ private function injectValuesForCustomLayout(?array &$layout): void
      */
     public function getSelectOptions(Request $request): JsonResponse
     {
-        $objectId = $request->query->getInt('objectId');
+        $objectId = (int)$request->get('objectId');
```
Comment (blankse) [inline src/Controller/Admin/DataObject/DataObjectController.php]: The get method is marked as internal and shouldn't be used anymore.
```suggestion
        $objectId = $request->request->getInt('objectId');
```

**Comment by mhgtlc** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by dvesh3** [comment]
@mhgtlc thanks!

**Pull request merged**